Sources/OpenAPIRuntime/Conversion/Configuration.swift

```diff
@@ -25,14 +25,45 @@ public protocol DateTranscoder: Sendable {
 }
 
 /// A transcoder for dates encoded as an ISO-8601 string (in RFC 3339 format).
-public struct ISO8601DateTranscoder: DateTranscoder {
+public struct ISO8601DateTranscoder: DateTranscoder, @unchecked Sendable {
+
+    /// The lock protecting the formatter.
+    private let lock: NSLock
+
+    /// The underlying date formatter.
+    private let locked_formatter: ISO8601DateFormatter
+
+    /// Creates a new default transcoder.
+    public init() { self.init(optionsOverride: nil) }
+
+    /// Creates a new transcoder with the provided options.
+    /// - Parameter optionsOverride: Options to override the default ones. If you provide
+    ///   nil here, the default options are used.
+    public init(optionsOverride: ISO8601DateFormatter.Options?) {
```

Comment:
Looking for feedback on this parameter here. Not happy with it yet.

Reply:
Maybe if we spell this as `init(options: ISO8601DateFormatter.Options? = nil)`, the optionality and nil-default will imply a default-when-nil, which can also be documented for additional clarity.

Reply:
Yeah I like that better, thanks.